aztec/src/main/kotlin/org/wordpress/aztec/ime/EditorInfoUtils.kt

```diff
@@ -0,0 +1,55 @@
+package org.wordpress.aztec.ime
+
+import android.os.Build
+import android.view.inputmethod.EditorInfo
+import java.util.Arrays
+
+object EditorInfoUtils {
+    @JvmStatic
+    fun areEditorInfosTheSame(ed1: EditorInfo, ed2: EditorInfo): Boolean {
+        if (ed1 == ed2) {
+            return true
+        }
+
+        if (ed1.actionId == ed2.actionId
```

Comment:
np: Perhaps some comments here can be helpful to provide context on this comparison